portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,85 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin: 20px;
 }
+#nav_bar{
+    display: flex;
```

Comment:
[2.2.1](https://google.github.io/styleguide/htmlcssguide.html#Indentation): Indent by 2 spaces at a time.

You have the appropriate indentation here and in the very last CSS rule _but_ not in the rest of the file.

For some reason the Cloud Shell Editor does not currently support formatting CSS files though so the easiest way to do this is by:
1. Selecting the entire document **except** for the correctly indented first and last CSS rules.
1. Outdenting with `Shift` + `Tab` or `Ctrl`/`Cmd` + `[`.